src/devhub/page/create.jsx

```diff
@@ -10,7 +10,7 @@ const AutoComplete = styled.div`
 `;
 
 function textareaInputHandler(value) {
-  const showAccountAutocomplete = /@[\w][^\s]*$/.test(value);
+  const showAccountAutocomplete = /(^|\s)@/.test(value);
```

Comment:
Will this make autocomplete bar always showup, as `@` will always exist?

Reply:
no actually not
it works when there is a whitespace before @ or if it's in the beginning of the text